netops_demo/README.md

```diff
@@ -157,6 +166,22 @@ We can now start the generation:
 http localhost:<function port>/start
 ```
 
+By default, the predict function is idling - waiting for configuration. Let's configure it by POSTing the following configuration to `/configure`:
+```
+echo '
+{
+  'metrics': <metrics array>,
+  'tsdb': <tsdb server url:proemtheus port>,
+  'state': 'predicting',
```

Comment:
1. Why are you setting the state here?
2. Rather than `tsdb` which is highly ambiguous, use `prometheus_url`